utilities/database.py

```diff
@@ -35,7 +35,7 @@ async def selector(query: str, variables: list):
     return result
 
 
-async def modifier(query: str, variables: list) -> None:
+async def modifier(query: str, variables: list):
```

Comment:
```suggestion
async def modifier(query: str, variables: list) -> None:
```
It does in fact return nothing. You can keep this.

Reply:
I looked at the function above, which also has no return value. Should I give that one too then?

Reply:
Yes.

Reply:
Well, the return type turns out to be quite messed up and I can't really make that happen on second thought. It'd be extremely generic anyway, so I don't think it's necessary anyway; it depends on the request made and you should roughly expect what kind of data you're getting anyway

Reply:
You can just dedicate `selector` as returning a tuple.
It's fine to leave typehinting out with mysql_login, as we cannot verify the return type of that well.